src/workerd/api/node/zlib-util.h

```diff
@@ -1,9 +1,16 @@
 // Copyright (c) 2017-2022 Cloudflare, Inc.
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
-#include <kj/array.h>
+
 #include <workerd/jsg/jsg.h>
 
+#include <kj/array.h>
+#include <kj/compat/brotli.h>
+#include <zlib.h>
```

Comment:
This is probably why your internal builds are failing as well:

```suggestion
#include "zlib.h"
```

Reply:
Unfortunately does not work.